src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    private List<Excavator> list = new ArrayList<>();
+
+    public ExcavatorProducer() {
+        list.add(new Excavator(true, "V 2.3"));
+        list.add(new Excavator(false, "V 3.0"));
+    }
+
+    @Override
+    public List<Excavator> get() {
+        return list;
```

Comment:
producer should create new methods each time when we call `get` method, rather creating them once in constructor

Reply:
updated